cassandra-diagnostics-commons/pom.xml

```diff
@@ -4,9 +4,30 @@
     <parent>
         <groupId>io.smartcat</groupId>
         <artifactId>cassandra-diagnostics</artifactId>
-        <version>1.0.0</version>
+        <version>1.0.0-SNAPSHOT</version>
     </parent>
     <artifactId>cassandra-diagnostics-commons</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <source.level>1.7</source.level>
+        <code.level>1.7</code.level>
+        <version.plugin.compiler>3.5.1</version.plugin.compiler>
+    </properties>
+      
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Can we postpone the suggested modification for an upcoming PR that will address the project organization (modules, dependencies, etc.)? This PR is primarily about refactoring Query class.